### PR TITLE
John conroy/sample metadata

### LINKED ISSUE #536: Show sample metadata fields on details

### LINKED ISSUE #737: Fix metadata table to handle boolean values

### PR DESCRIPTION
**Pull request by john-conroy**
Fixes #536. Fixes #737 .

### CODE REVIEW

**Comment by mccalluc** [comment]
@john-conroy : When this is ready, add me as reviewer?

File: context/app/static/js/components/Detail/Sample/Sample.jsx [hunk 1]
```diff
@@ -4,9 +4,10 @@ import ProvTabs from '../ProvTabs';
 import Summary from '../Summary';
 import Attribution from '../Attribution';
 import Protocol from '../Protocol';
-import Metadata from '../Metadata';
 import SummaryItem from '../SummaryItem';
 import DetailLayout from '../DetailLayout';
+import MetadataTable from '../MetadataTable/MetadataTable';
```
[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/Detail/Sample/Sample.jsx:9]: This was just a mistake. I believe vs-code auto-imported.

[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/Detail/Sample/Sample.jsx]: We can talk tomorrow. Should I still hold off on merging given the correction?

[the remark this answers is not in the document]
Comment (mccalluc) [reply context/app/static/js/components/Detail/Sample/Sample.jsx]: Latest commit looks fine. Sorry to be picky! Resolve conflicts and merge at your convenience.

**Pull request merged**